planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp

```diff
@@ -85,7 +85,7 @@ std::vector<int64_t> replaceWithSortedIds(
 std::vector<std::vector<int64_t>> get_sorted_lane_ids(const CommonDataPtr & common_data_ptr);
 
 lanelet::ConstLanelets getTargetNeighborLanes(
```

Comment:
change to use snake_case?

Reply:
done in https://github.com/autowarefoundation/autoware.universe/pull/9085/commits/46cf31901739f76a0b699edd9b542012d140611e